operator/pkg/ciliumendpointslice/rate_limit.go

```diff
@@ -29,87 +30,68 @@ type rateLimitConfig struct {
 	logger logrus.FieldLogger
 }
 
+type dynamicRateLimit []rateLimit
+
+var defaultDynamicRateLimit = dynamicRateLimit{
+	{Nodes: 0, Limit: 10, Burst: 20},
+	{Nodes: 100, Limit: 7, Burst: 15},
+	{Nodes: 500, Limit: 5, Burst: 10},
+}
```

Comment:
I personally find a complex default configuration like this to be possibly confusing for users, as there's no easy way to figure it out except looking at the code. How about having a single limit/burst default, and then overriding it by default through the helm values? That way, the default values would be immediately visible to the users, and tuning would also be way simpler. That approach would also allow to drop the additional multiplier flag introduced in the second commit, as the values could be tuned directly if necessary.

Reply:
SGTM :+1: I think that's a more user-friendly approach :+1: